engine/src/main/java/org/openobservatory/engine/OONISessionConfig.java

```diff
@@ -39,7 +39,7 @@ public final class OONISessionConfig {
      * usage of an alternative probe services. This setting
      * should only be used to implement integration tests.
      */
-    String probeServicesURL;
+    public String probeServicesURL;
```

Comment:
Unless I'm missing something, the probe services URL is always passed as an argument and therefore we can back out this specific change.

```suggestion
    String probeServicesURL;
```